DataManagementSystem/Agent/RequestOperations/RegisterReplica.py

```diff
@@ -95,9 +96,8 @@ def __call__(self):
               catMaster = catMaster and FileCatalog()._getCatalogConfigDetails(failedCatalog)\
                   .get('Value', {}).get('Master', False)
           # If one targets explicitly a catalog and it fails or if it fails on the master catalog
-          if (
-                  catalogs or catMaster) and (
-                  'file does not exist' in opFile.Error.lower() or 'no such file' in opFile.Error.lower()):
+          if (catalogs or catMaster) and \
+                  ('file does not exist' in opFile.Error.lower() or 'no such file' in opFile.Error.lower()):
```

Comment:
A minor comment on PEP8: it's suggested to break the line after the binary operator:
python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

Reply:
You mean BEFORE?